content/tutorial/tutorial.md

```diff
@@ -12,185 +12,190 @@ redirect_from:
-  React.createElement('h1', /* ... h1 children ... */),
-  React.createElement('ul', /* ... ul children ... */)
+  React.createElement('h1', /* ... h1 चिल्ड्रन ... */),
+  React.createElement('ul', /* ... ul चिल्ड्रन ... */)
 );
 ```
 
-[See full expanded version.](babel://tutorial-expanded-version)
+[पूर्ण विस्तारित संस्करण देखें।](babel://tutorial-expanded-version)
 
-If you're curious, `createElement()` is described in more detail in the [API reference](/docs/react-api.html#createelement), but we won't be using it in this tutorial. Instead, we will keep using JSX.
+यदि आप उत्सुक हैं, तो [API संदर्भ](/docs/react-api.html#createelement) में `createElement()` को अधिक वर्णन के साथ बताया गया है, लेकिन हम इस ट्यूटोरियल में इसका उपयोग नहीं करेंगे। इसके बजाय, हम JSX का उपयोग करते रहेंगे।
 
-JSX comes with the full power of JavaScript. You can put *any* JavaScript expressions within braces inside JSX. Each React element is a JavaScript object that you can store in a variable or pass around in your program.
+JSX जावास्क्रिप्ट की पूरी शक्ति के साथ आता है। आप JSX के अंदर ब्रेसिज़ के भीतर *कोई* जावास्क्रिप्ट एक्सप्रेशन रख सकते हैं। प्रत्येक React एलिमेंट एक जावास्क्रिप्ट ऑब्जेक्ट है जिसे आप एक वैरिएबल् में स्टोर कर सकते हैं या अपने प्रोग्राम में कहीं भी पास कर सकते हैं।
 
-The `ShoppingList` component above only renders built-in DOM components like `<div />` and `<li />`. But you can compose and render custom React components too. For example, we can now refer to the whole shopping list by writing `<ShoppingList />`. Each React component is encapsulated and can operate independently; this allows you to build complex UIs from simple components.
+`ShoppingList` कौम्पोनॅन्ट ऊपर केवल बिल्ट-इन DOM जैसे `<div />` और `<li />` को रेंडर करता है। लेकिन आप कस्टम React कौम्पोनॅन्ट्स को भी कम्पोज़ और रेंडर कर सकते हैं। उदाहरण के लिए, हम अब पूरी खरीदारी सूची को `<ShoppingList />` लिखकर संदर्भित कर सकते हैं। प्रत्येक React कौम्पोनॅन्ट संकुचित है और स्वतंत्र रूप से काम कर सकता है; यह आपको सरल कौम्पोनॅन्टस से कॉम्प्लेक्स UI बनाने की अनुमति देता है।
 
-## Inspecting the Starter Code {#inspecting-the-starter-code}
+## स्टार्टर कोड का निरीक्षण {#inspecting-the-starter-code}
 
-If you're going to work on the tutorial **in your browser,** open this code in a new tab: **[Starter Code](https://codepen.io/gaearon/pen/oWWQNa?editors=0010)**. If you're going to work on the tutorial **locally,** instead open `src/index.js` in your project folder (you have already touched this file during the [setup](#setup-option-2-local-development-environment)).
+यदि आप **अपने ब्राउज़र में** ट्यूटोरियल पर काम करने जा रहे हैं, इस कोड को एक नए टैब में खोलें: **[स्टार्टर कोड।](https://codepen.io/gaearon/pen/oWWQNa?editors=0010)**. यदि आप **लोकली** रूप से ट्यूटोरियल पर काम करने जा रहे हैं, तो अपने प्रोजेक्ट फ़ोल्डर में `src/index.js` खोलें ([सेटअप](#setup-option-2-local-development-environment) के दौरान आप इस फ़ाइल को पहले ही देख चुके हैं)।
```

Comment:
by full stop I meant purn viraam :laughing:

Reply:
तो अपने प्रोजेक्ट फ़ोल्डर में `src/index.js` खोलें <insert purn viraam here as well>

Reply:
> by full stop I meant purn viraam 😆

`खोलें ([सेटअप](#setup-option-2-local-development-environment) के दौरान आप इस फ़ाइल को पहले ही देख चुके हैं)।`
this will look like this after rendering

खोलें ([सेटअप](#setup-option-2-local-development-environment) के दौरान आप इस फ़ाइल को पहले ही देख चुके हैं)।

Which has purn viraam at the end after the bracket, if that's what you meant

Reply:
looks okay I guess